webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -81,7 +77,6 @@ const ImportButton = ({ config, repo, reference, path, onDone, onClick, variant
     const sourceRef = useRef(null);
     const destRef = useRef(null);
     const commitMsgRef = useRef(null);
-    const storageType = config.blockstore_type
     const storageNamespaceValidityRegexStr = config.blockstore_namespace_ValidityRegex;
     const storageNamespaceValidityRegex = RegExp(storageNamespaceValidityRegexStr);
     const sourceURIExample = config ? config.blockstore_namespace_example : "s3://my-bucket/path/";
```

Comment:
Lines 86-90 are wrong, think about importing to a branch named `main_2`

Reply:
Opened #3416 for it, we should address it there